sensor/kubernetes/listener/resource_event_handler.go

```diff
@@ -294,6 +307,11 @@ func handle(
 		missingInitialIDs:          nil,
 	}
 	informer.AddEventHandler(handlerImpl)
+	if !informer.HasSynced() {
+		if err := informer.SetTransform(managedFieldsTransformer); err != nil {
+			utils.Should(err)
```

Comment:
There is no need to guard `utils.Should` with condition. If err is nil it will do nothing.